### PR TITLE
Dutch translation corrections

### LINKED ISSUE #3000: Corrected Dutch translations were not updated in NPSP 3.120
[NPSP 3.120](https://github.com/SalesforceFoundation/Cumulus/releases/tag/rel%2F3.120) contained a refresh of corrected Dutch translations, however it appears that prior incorrect values were not replaced as expected.

### PR DESCRIPTION
**Pull request by jjbennett**
// There are translation marker updates for new fields and labels as well as the dutch translation fixes.

# Critical Changes

# Changes

# Issues Closed
Fixes #3000

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by jjbennett** [comment]
@joy-sfdo this shouldn't be merged until after the release freeze from this sprint is lifted. Thanks again!

**Comment by sam-knox** [comment]
@joy-sfdo - Can you summarize what the changes are in the Changes section in the comments?

**Comment by jjbennett** [comment]
@sam-knox - There is an issue being closed with this PR, do we need a changes as well?

**Comment by sam-knox** [comment]
@jjbennett - I asked Sarah about this actually. We want to thank those who helped with the translations. We're tracking down who they were. The change note will read something like:

"We made some updates to the Dutch translations. Special thanks to . . . "

So, @joy-sfdo, I don't think you need to do anything afterall.
